src/applications/simple-forms-form-engine/shared/utils/startApp.js

```diff
@@ -0,0 +1,28 @@
+import '@department-of-veterans-affairs/platform-polyfills';
+import '../sass/simple-forms-form-engine.scss';
+
+import React from 'react';
+import { startAppFromIndex } from '@department-of-veterans-affairs/platform-startup/exports';
+import reducer from '../reducers';
+import FormRenderer from '../containers/FormRenderer';
+import { removeTrailingSlash } from './string';
+
+export const startFormEngineFormApp = ({
```

Comment:
I'm not in love with this name, but I don't have any better suggestions. Do we need the second `Form`, or can we assume any `FormEngineApp` will be a form?

Reply:
I can buy this. I think it's technically "correct" as written - and I tend to like correctness over brevity, personally - but I do think it could be considered correct to just have `startFormEngineApp`. To your point, all apps tied to the Form Engine should be forms. So, under that assumption, the latter option is correct while being more brief. I'm fine to switch it to that.

Fwiw, I also considered changing this altogether as a result of the verbose naming. It occurred to me that we don't actually need to name this anything other than `startApp`, because the directory structure already provides context.

Reply:
This is not a blocker, so do whatever you feel is best for the moment. We can always revisit it later, but I wanted to make sure we discussed it now before it starts getting imported into a bunch of Form Builder apps.

Reply:
I'm going to change this. I agree it's more wordy than necessary.